crates/pera-core/src/authority.rs

```diff
@@ -1522,6 +1531,41 @@ impl AuthorityState {
         Ok(())
     }
 
+    /// Handle the MPC signature events emitted from the transaction, if any.
+    /// The filtering to handle only signature-mpc related events happens within [`SignatureMPCManager::handle_mpc_events`] function.
+    async fn handle_signature_mpc_events(
```

Comment:
This is confusing; the `Authority` has a `handle_signature_mpc_events` function, but the `SignatureMPCManager` has a
`handle_mpc_events` function?